src/main/scala/wdlTools/types/Unification.scala

```diff
@@ -188,6 +184,7 @@ case class Unification(conf: Options) {
   //    T_Var(0) -> T_Int
   //    T_Var(1) -> T_String
   //
+  @throws(classOf[TypeUnificationException])
```

Comment:
If you want to use this kind of decoration please add a comment in a central location that we don't use it throughout the code in a rigorous manner. This is important when someone reads a method that *does not* have a `throws` decoration. It means that it *may* throw an exception, the omission is not a guarantee that no exception will be thrown.

Reply:
oops - left that there by mistake